platforms/core-configuration/model-core/src/main/java/org/gradle/internal/model/CalculatedValue.java

```diff
@@ -22,9 +22,13 @@
 
 /**
  * Represents a calculated immutable value that is calculated once and then consumed by multiple threads.
+ * <p>
+ * This class can hold a null value.
  */
 @ThreadSafe
 public interface CalculatedValue<T> {
+    // TODO(https://github.com/gradle/gradle/issues/24767): with JSpecify, the nullable nature of the type argument <T> should be expressed as <T extends @Nullable Object>.
```

Comment:
❓  Is this different from:

```
public interface Transformer<OUT extends @org.jetbrains.annotations.Nullable Object, IN> {
```

Is it that we'd rather not use JB's `Nullable` annotation?

Reply:
I guess you explain it here.

https://github.com/gradle/gradle/pull/28822/files#diff-575049641757d005928c369350bc3f33168e24dd0a00e208780998ce740f11dcR34

Reply:
>Is it that we'd rather not use JB's Nullable annotation?

There just too many false positive warnings in IDEA for this type and `Try` for it to be useful :(. Annotating client sides is enough for now - it shows the necessary warnings upon access, but not when passing `Try<@Nullable Foo>` argument to `Try<Foo>` parameter, unfortunately.